templates/Silk.NET.Templates/templates/silkgltriangle/Program.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using Silk.NET.Windowing;
+using Silk.NET.OpenGL;
+
+WindowOptions windowOptions = WindowOptions.Default with {
+    Title = "My Silk.NET Triangle",
+    PreferredDepthBufferBits = 0,
+    PreferredStencilBufferBits = 0,
```

Comment:
Might not be a bad idea to put them in all the templates actually, there's a reason why I joke about some drivers making your screen explode if you don't set the preferred depth bits.

It reminds people "If you wanna do 3D, you gotta set these" instead of sending them into a rabbit hole through forums, and it makes your app work as you'd expect in any system

Reply:
agree

Reply:
*To clarify: I personally feel we should set these to some sensible values like 24/8, so even if they modify the template for 3D they don't need to modify this bit

Reply:
Working Group Decided: Use 24/8 in templates

(we'll also accept a change in Silk.NET.Windowing.Glfw to make this default, rather than `GLFW_DONT_CARE`)